include/aikido/control/KinematicSimulationJointCommandExecutor.hpp

```diff
@@ -73,7 +86,9 @@ using KinematicSimulationPositionExecutor
     = KinematicSimulationJointCommandExecutor<ExecutorType::POSITION>;
 using KinematicSimulationVelocityExecutor
     = KinematicSimulationJointCommandExecutor<ExecutorType::VELOCITY>;
-// Note: No effort simulation without dynamics
+// Note: No effort simulation without dynamics, treat like velocity.
```

Comment:
Should we even have a KinematicSimulationEffortExecutor? How do we envision the codebase to change when we integrate with a dynamics simulator?

Reply:
Currently dynamics simulator is WONTFIX. This should be here primarily for hotswap compatibility between real and sim. i.e., if real has an effort executor, it should still be able to run in a Kinematic Simulation, even if it isn't perfect.